cases/query/simple_query.yaml

```diff
@@ -463,3 +463,297 @@ cases:
       columns: ["col1:bool"]
       rows:
         - [true]
+
+  - id: like_udf_1
+    desc: precent character
+    inputs:
+      - name: t1
+        columns: ["col1:int32", "std_ts:timestamp", "col2:string"]
+        indexs: ["index1:col1:std_ts"]
+        rows:
+          - [1, 1590115420001, "Mike"]
+    dataProvider:
+      - ["like_match", "ilike_match"]
+      - ["M%", "m%"]
+    sql: |
+      select d[0](col2, 'd[1]', '\\')  as col2 from t1;
+    expect:
+      columns: ["col2:bool"]
+    expectProvider:
+      0:
+        0:
+          rows:
+            - [true]
+        1:
+          rows:
+            - [false]
+      1:
+        0:
+          rows:
+            - [true]
+        1:
+          rows:
+            - [true]
+  - id: like_udf_2
+    desc: underscore character, 2 pararmeter version
+    inputs:
+      - name: t1
+        columns: ["col1:int32", "std_ts:timestamp", "col2:string"]
+        indexs: ["index1:col1:std_ts"]
+        rows:
+          - [1, 1590115420001, "Mike"]
+    dataProvider:
+      - ["like_match", "ilike_match"]
+      - ["M_ke", "m_ke"]
+    sql: |
+      select d[0](col2, 'd[1]')  as col2 from t1;
+    expect:
+      columns: ["col2:bool"]
+    expectProvider:
+      0:
+        0:
+          rows:
+            - [true]
+        1:
+          rows:
+            - [false]
+      1:
+        0:
+          rows:
+            - [true]
+        1:
+          rows:
+            - [true]
+  - id: like_udf_3
+    desc: disable escape
+    inputs:
+      - name: t1
+        columns: ["col1:int32", "std_ts:timestamp", "col2:string"]
+        indexs: ["index1:col1:std_ts"]
+        rows:
+          - [1, 1590115420001, "Mike"]
+    dataProvider:
+      - ["like_match", "ilike_match"]
+      - ["''" ]
```

Comment:
string(null)

Reply:
There might be a crush for `string(null)`. https://github.com/4paradigm/OpenMLDB/issues/854.

Reply:
Will support when #855 is merged